accumulo/src/main/java/org/vertexium/accumulo/AccumuloElement.java

```diff
@@ -230,16 +230,9 @@ public Property getProperty(String key, String name) {
 
     @Override
     public Property getProperty(String name) {
-        if (ID_PROPERTY_NAME.equals(name)) {
-            return getIdProperty();
-        } else if (Edge.LABEL_PROPERTY_NAME.equals(name) && this instanceof Edge) {
-            return getEdgeLabelProperty();
-        } else if (Edge.OUT_VERTEX_ID_PROPERTY_NAME.equals(name) && this instanceof Edge) {
-            return getOutVertexIdProperty();
-        } else if (Edge.IN_VERTEX_ID_PROPERTY_NAME.equals(name) && this instanceof Edge) {
-            return getInVertexIdProperty();
-        } else if (Edge.IN_OR_OUT_VERTEX_ID_PROPERTY_NAME.equals(name) && this instanceof Edge) {
-            return getInOrOutVertexIdProperty();
+        Property reservedProperty = getReservedProperty(name);
```

Comment:
This check for reservedProperty will be taken care of in the call to `getProperties(name)` below.